app/src/main/java/org/mozilla/fenix/settings/account/TurnOnSyncFragment.kt

```diff
@@ -89,6 +100,21 @@ class TurnOnSyncFragment : Fragment(), AccountObserver {
             getString(R.string.sign_in_instructions),
             HtmlCompat.FROM_HTML_MODE_LEGACY
         )
+
+        val createAccountActionText = getString(R.string.sign_in_create_account_link)
+        val fullText = getString(R.string.sign_in_create_account_text, createAccountActionText)
+        val spanStart = fullText.indexOf(createAccountActionText, 0, false)
+        val spanEnd = spanStart + createAccountActionText.length
+
+        view.createAccount.apply {
+            text = fullText
+            addUnderline(
+                spanStart,
+                spanEnd,
+                Spanned.SPAN_EXCLUSIVE_EXCLUSIVE
+            )
+            setOnClickListener(createAccountClickListener)
```

Comment:
nit : could we just do this here?
```suggestion
            setOnClickListener { navigateToPairWithEmail() }
```

Reply:
We could do that, but I intentionally made this like that, so any future change would be made in `createAccountClickListener` not inside the click listener setter. 
Also,  it is more easily readable that it is not a mistake, setting the same action on 2 different views.